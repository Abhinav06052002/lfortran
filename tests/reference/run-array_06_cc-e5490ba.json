{
    "basename": "run-array_06_cc-e5490ba",
    "cmd": "lfortran --continue-compilation --no-color {infile}",
    "infile": "tests/errors/array_06_cc.f90",
    "infile_hash": "a2934ed73b27bb33b6f812f17615780a23513b88cf68a4dcc98f0f72",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "run-array_06_cc-e5490ba.stdout",
    "stdout_hash": "a779ee4f7341ddce58a41cee3d249ecf1b37216d7d4f16eb4e0f528e",
    "stderr": "run-array_06_cc-e5490ba.stderr",
    "stderr_hash": "40279d827c309a6f1a8612615495a233cbb6f1d158a6b3f442710569",
    "returncode": 0
}