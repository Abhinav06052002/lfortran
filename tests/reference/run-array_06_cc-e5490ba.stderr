semantic error: Rank mismatch in array reference: the array `str` has rank `1, but is referenced as rank `2
 --> tests/errors/array_06_cc.f90:5:5
  |
5 |     str(1, 2)(:) = '1234'
  |     ^^^^^^^^^^^^ 

semantic error: Rank mismatch in array reference: the array `str` has rank `1, but is referenced as rank `3
 --> tests/errors/array_06_cc.f90:6:5
  |
6 |     str(1,2,3)(:) = '1234'
  |     ^^^^^^^^^^^^^ 
